### PR TITLE
Entity registration number

### PR DESCRIPTION
**Pull request by xacobofg**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

Add administrative number field for entities and allow filtering it
imagen

### CODE REVIEW

**Comment by OscarBeiro** [comment]
Hi,
@cedric-anne Could you please lend us a hand on why the test PHP 8 Mariadb:10.6 is failing?
Error 1 is very general. Any tip is appreciated.
Thanks

File: inc/update.class.php [hunk 1]
```diff
@@ -496,6 +496,11 @@ public function doUpdates($current_version = null) {
             update955to956();
             break;
```
Comment (cedric-anne) [inline inc/update.class.php:497]: Next upgrade will not be done if made from a version prior to 9.5.6.
```suggestion
```

Comment (xacobofg) [reply inc/update.class.php]: Hi.
Thanks for the tip, the tests are now working.

**Comment by OscarBeiro** [comment]
Merci @cedric-anne :)

**Comment by orthagh** [comment]
Hello @xacobofg and @OscarBeiro 

We're ok with this pr but not the current target.
- 9.5/bugfixes should only have corrective commits as it's a maintenance branch
- master is the current development branch and the perfect target for your changes.

This retargetting should not be complex, you must just change your migration name i think. The UI part is not (yet) currently impact by modern-ui.
(Wait 1 or 2 hours, we are currently merging modern-ui in master)

Regards

**Comment by xacobofg** [comment]
Hi @cedric-anne and @orthagh 
Sorry for the delay, I updated the branch to target master.

**Review by trasher — APPROVED** [review]

**Review by orthagh — APPROVED** [review]

**Pull request merged**